Anlab.Core/Data/ApplicationDbContext.cs

```diff
@@ -96,6 +98,10 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<ReviewerOrderView>()
                 .ToTable("ReviewerOrders");
 
+            //Probably don't need this here because the name is the same as the table name
+            builder.Entity<HistoricalSalesView>()
+                .ToTable("HistoricalSalesView");

```

Comment:
You can (should??) use `.ToView()` here.  I'm not sure what it does in practice, but I assume it keeps an internal mapping that is read/query only.